pytest_fixtures/component/satellite_auth.py

```diff
@@ -289,7 +289,8 @@ def auth_data(request, ad_data, ipa_data):
 @pytest.fixture(scope='module')
 def enroll_configure_rhsso_external_auth(module_target_sat):
     """Enroll the Satellite6 Server to an RHSSO Server."""
-    module_target_sat.register_to_cdn()
+    if settings.robottelo.rhel_source == "ga":
```

Comment:
Haven't seen this in `robottelo.yaml` in Jenkins (though it's present in `robottelo.yaml.template`).
Do I assume correctly that we depend purely on the validator [here](https://github.com/SatelliteQE/robottelo/blob/57a4ee04416108eac37411d3265747204e9a6c8b/robottelo/config/validators.py#L326)?

Maybe rather question for @rmynar.

Reply:
I think we can try available setting `settings.robottelo.cdn == true` here?

Reply:
I would not say we have to "depend" on validators. But the idea is to maintain validators responsibly (not only in this case) so we can "trust" them.
Relying on default values and modifying the settings only in specific cases seems OK to me. (The _rhel_source_ setting has to be explicitly changed to "internal" usually for interoperability testing - i.e. testing with unreleased RHEL version)

Reply:
The setting is set in the `.template` config https://github.com/SatelliteQE/robottelo/blob/b7406c0ff21fb5d696abbf6e15725b21d9c8dd6d/conf/robottelo.yaml.template#L25-L27.

And yes, in this case, we rely on Dynaconf validators to populate the setting for us, but since validators run unconditionally every time (now even in Robottelo CI/CQ), I share the same view as @rmynar. However, there is no harm in adding an explicit setting in the YAML config. Anyone, feel free to submit a patch.